src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Webservice/index.html.twig

```diff
@@ -52,25 +52,65 @@
     {% endif %}
   {% endblock %}
 
+  {% block webservice_api_status %}
+    {% if webserviceStatus['enabled'] == true %}
+      <div class="card">
+        <h3 class="card-header">
+          <i class="material-icons">info_outline</i> {{ 'Webservice status'|trans }}
+        </h3>
+        <div class="card-block">
+          <div class="card-text">
+            <p>
+              Webservice is enabled. Main entry point is
```

Comment:
Need some translations tags

Reply:
I know, I wrote it before 😄
#19335 (comment)

As I have not been provided feedback by PM on this new (and unscoped) feature I wait to get approval/specs before doing the polishing. Because it's useless to make the code spotless if I have to redo it 100% because PM ask me to do something different

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/19335/commits/db6866eec2cf932f050ef74981bd353c4e967f52